src/components/organisms/events/events.scss

```
@@ -0,0 +1,86 @@
+@use "../../variables" as *;
+
+@mixin events-picture-hover {
+  transform: scale(1.2, 1.2);
+}
+@mixin events-img {
+  height: 141px;
+  background-size: cover;
+}
+
+.events {
+  .english {
+    font-weight: normal;
+    display: inline-block;
+    vertical-align: middle;
+    margin-right: 16px;
+    font-size: $fontsize-l;
+    font-family: $fontfamily-mincho;
+    @media (min-width: $breakpoint-lg) {
+      font-size: $fontsize-xxl;
+    }
+  }
+  .japanese {
+    vertical-align: middle;
+    font-size: $fontsize-m;
+  }
+}
+.events-href {
+  text-decoration: none !important;
```

importantよりはこちらの方がいいかと思います。
```suggestion
.events-href {
 &:hover {
  text-decoration: none;
 }
```